### DISCUSSION TITLE
Fancybox 4 Static Backdrop

### DISCUSSION THREAD

**Discussion by b-s-j** [fancyapps/ui#373] [Q&A]
Please help me how to make a static backdrop when clicking outside fancybox 4 it will not close instead it allows me to click next button on datatables inside my fancybox ?

**Comment by fancyapps** [comment]
Hi,

Sorry, but I don't understand your question, could you please clarify what do you mean by `static backdrop`? Also, any demo would help better understanding.

**Reply by pbern2** [reply]
yeah, thank you for the response ok so if the fancy box shows a white color box appears you can also click outside the box to close it right? But what I need is if I click outside the fancy box it will not close. by default fancybox will provide a close button on the upper right corner

**Reply by fancyapps** [reply]
Looks like you are looking for `click` option, set `click:false` and Fancybox will not close when clicking outside the content.

**Reply by b-s-j** [reply]
thank you sir you are correct click:false
